src/components/Header/Header.jsx

```diff
@@ -150,14 +150,14 @@ class Header extends React.Component {
       </VerticalAlignToolbar>
     );
 
-    const SettingsGroup = ({ user }) => (
+    const SettingsGroup = ({ userSettings = user }) => (
```

Comment:
remove default value, same check for break

Reply:
Same as other warning in same file, reverting until figured out.